### ISSUE TITLE
String width limit

### ISSUE THREAD

**Issue by pimlie** [SheetJS/printj#4]
It seems that when setting a string width the printed width could be lower then specified when a short input string is used.

For example:
```bash
$ printj '|%1$60s|' s:"hello" | wc -c
41
$ printj '|%1$60s|' s:"hello world, how are you ?" | wc -c
62
$ printj '|%1$60s|' s:"hello world, how are you ??" | wc -c
63
```
I would expect that the first and second statements would also return 63 characters?

--edit--
It seems thats because the padstr's are limited to 20 chars and there are 2 iterations? Could we maybe lengthen the pad strings a bit?

Or maybe even use String.repeat instead with a [polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat)?

**Comment by SheetJSDev** [comment]
Just following up -- we published 1.2.1 with the fix prescribed in #5 and the results are consistent now.  Thank you very much!

**Comment by pimlie** [comment]
Great, thank you for the quick merge!
